subworkflows/nf-core/abundance_differential_filter/tests/main.nf.test

```diff
@@ -219,7 +219,8 @@ nextflow_workflow {
             assertAll(
                 { assert workflow.success },
                 { assert snapshot(
-                    path(workflow.out.results_genewise[0][1]).exists(),
+                    file(workflow.out.results_genewise[0][1]).name,
+                    path(workflow.out.results_genewise[0][1]).getText().contains("1007_s_at\t-0.2775254"),
```

Comment:
Close, but this should actually be a separate assertion, outside of the snapshotting.